### PR TITLE
Fix CL2.0 sub flow

### PR DESCRIPTION
**Pull request by nwingt**
Depends on https://github.com/likecoin/like-co/pull/1489

### CODE REVIEW

**Review by williamchong — APPROVED** [review]

File: src/components/CivicSubscriptionView/CivicSubscriptionView.vue [hunk 1]
```diff
@@ -469,8 +511,6 @@ export default {
       try {
         await Promise.all(promises);
       } catch (err) {
-        // eslint-disable-next-line no-console
-        console.error(err);
```
Comment (williamchong) [inline src/components/CivicSubscriptionView/CivicSubscriptionView.vue:473]: Why remove, do we expect any error now?

File: src/util/api/index.js [hunk 2]
```diff
@@ -61,9 +62,20 @@ export const getUpdateReaderBookmarkAPI = url =>
...
 export const getFetchFollowedSuperLikeApi = () =>
   `/api/reader/superlike/followed`;
-export const getOAuthRegisterAPI = (from = '', referrer = '') =>
-  `/api/users/register?from=${from}&referrer=${encodeURIComponent(referrer)}`;
-export const getOAuthLoginAPI = () => '/api/users/login';
+export const getOAuthRegisterAPI = (
+  language = 'zh',
+  from = '',
+  referrer = ''
+) =>
```
Comment (williamchong) [inline src/util/api/index.js:69]: Would using named param better? These 3 params are all optional, and doesn't have a logical relationship that one must be placed earlier than the others.

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/CivicSubscriptionView/CivicSubscriptionView.vue (hunk 1 above)
Comment (nwingt) [reply src/components/CivicSubscriptionView/CivicSubscriptionView.vue]: Updated

File: src/util/api/index.js (hunk 2 above)
Comment (nwingt) [reply src/util/api/index.js]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**